Gemfile.rails_version

```diff
@@ -2,7 +2,7 @@
 # separate fragment so that it can be sourced from the test
 # application's Gemfile in addition to the main development Gemfile.
 
-env_rails_version = ENV['RAILS_VERSION'] || '5.1.7'
+env_rails_version = ENV['RAILS_VERSION'] || '5.2'
```

Comment:
Is 5.2 a valid version? Wouldn't it be `5.2.0`?

Reply:
well, its used in a `~>` so its technically for anything < 6 but I can go back to validating only for 5.2